apps/frontend/src/components/global/login/LocalLogin.vue

```diff
@@ -25,13 +25,18 @@
           ref="password"
           v-model="password"
           :error-messages="requiredFieldError($v.password, 'Password')"
-          type="password"
+          :type="showPassword ? 'text' : 'password'"
           name="password"
           label="Password"
           prepend-icon="mdi-lock"
           tabindex="2"
           @blur="$v.password.$touch()"
-        />
+          ><template #append>
```

Comment:
formatting please - the automated stuff looks terrible

Reply:
Good catch.  I don't know why the linter isn't picking those up.  I know we had similar issues with the checklist PR stuff.  It seems like sometimes it catches it and other times it doesn't